os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/board/common/src/hci_adapter.c

```diff
@@ -175,16 +173,20 @@ void hci_tp_config(uint8_t *p_buf, uint16_t len)
 hci_tp_config_ok:
                 /* next step */
                 g_hci_step++;
+                //dbg("-- [hci_tp_config_ok]  %p %d--\n", __builtin_return_address(0), g_hci_step);
                 if(g_hci_step == hci_total_step)
                 {
                     hci_board_debug("\r\n%s:BT INIT success %x\n",__FUNCTION__,g_hci_step);
                     if(hci_board_complete() == true)
                     {
                          p_hci_rtk->open_cb(true);
+                         bte_init_buff_free();

```

Comment:
How about free after line 179? Because you called free both in true and false case.

Reply:
OK. I have mdified.